internal/orchestrator/orchestrator.go

```diff
@@ -67,7 +67,9 @@ var (
 	// For backward compatibility with the old "publicstorage" prefix
 	publicstorageConfig = config.NewPluginConfig("publicstorage")
 	dataexchangeConfig  = config.NewPluginConfig("dataexchange")
-	tokensConfig        = config.NewPluginConfig("tokens").Array()
+	// tokenConfig is deprecated in favor of tokensPluginConfig
```

Comment:
This comment confusingly doesn't match the variable names below. Might not be needed now that "deprecated" is in the var name?

Reply:
Good catch, thanks!